app/src/main/java/fr/free/nrw/commons/review/ReviewController.java

```diff
@@ -166,8 +168,7 @@ void sendThanks(@NonNull Activity activity) {
                         message = context.getString(R.string.send_thank_failure_message, media.getDisplayTitle());
                     }
 
-                    showNotification(title, message);
-
+                    ViewUtil.showShortToast(context,message);
```

Comment:
This statement was not covered in tests.
Add a test for this line ?

Reply:
@4D17Y4, how can i cover the code inside of the Observable.defer ?

Reply:
I believe it should be possible, can u look at the present instances in the codebase if there are any ? 
I think we have similar code but not sure if they are covered by tests. Let me know in any case.

Reply:
@madhurgupta10, can you please help in this ?

Reply:
I don't have access to my computer for a few days, maybe you can look at some samples in the codebase or maybe @devarsh-mavani-19 or @Ayan-10 can help you if they faced similar issues before.

Reply:
> I don't have access to my computer for a few days, maybe you can look at some samples in the codebase or maybe @devarsh-mavani-19 or @Ayan-10 can help you if they faced similar issues before.

I told him solution over call a few hours ago. As we were from same college previously and we are in touch with each other.

The way i handled it is by putting code of defer method in a separate function and writing tests for that function. i have implemented it previously in 1 of my PRs will update if i find it. Thanks